cpp/zotaut/adjust_selective_evaluation_validation_rules.cc

```diff
@@ -79,6 +79,9 @@ std::string GetJournalId(DbConnection * const db_connection, const std::string &
     db_connection->queryOrDie("SELECT id FROM zeder_journals WHERE zeder_id=\'" + zeder_id +
                               "\' AND zeder_instance=\'" + group + "\'");
     DbResultSet result_set(db_connection->getLastResultSet());
+    if (not result_set.size()) {
```

Comment:
Please do not treat integers as boolean values!

Reply:
Why don't you use empty instead of this abomination?

Reply:
Adjusted